osu.iOS/Info.plist

```diff
@@ -10,8 +10,6 @@
 	<string>osu!</string>
 	<key>CFBundleShortVersionString</key>
 	<string>0.1</string>
```

Comment:
This could also be removed and replaced with `ApplicationDisplayVersion`, similar to how it was done when switching Android to .NET 6.0 in https://github.com/ppy/osu/pull/21878.